src/components/common/AppHeader.vue

```diff
@@ -81,13 +81,13 @@ export default {
   data: () => ({
     open: false,
     desktop: false,
-    isMobileApp: config.mobileApp,
+    isMobileApp: config.mobileApp
```

Comment:
oops, look at that... the linting is still being different for each of us.

We didn't reach consensus yet on trailing comma yes or not

Reply:
I would rely on the standards which switched to trailing commas. Weird this isn't applied here.